src/lib/actions.js

```diff
@@ -229,11 +230,12 @@ export const setLanguage = store.action((state, language) => {
 });
 
 export const loadAnalyticsScript = store.action(() => {
-  const {g4ScriptLoaded} = store.getState();
-  if (!g4ScriptLoaded && isProd) {
-    loadScript(`https://www.googletagmanager.com/gtag/js?id=${ids.GA4}`, null);
+  const {gtmScriptLoaded} = store.getState();
+  if (!gtmScriptLoaded && !isProd) {
```

Comment:
Change this back to `isProd` before merging:

```suggestion
  if (!gtmScriptLoaded && isProd) {
```

Or alternatively we can remove this, now we have staging properly set up.